### DISCUSSION TITLE
1.2 beta 3 => 1.2 beta 4 project compatibility?

### DISCUSSION THREAD

**Discussion by igbro** [apohl79/audiogridder#679] [General]
Hi Andreas,

Loading a1.2 beta 3 project in Cubase with1.2 beta 4 server/plugin installed completely crashes the server machine system (not only the server app).

All Windows here, W10 Cubase 11 pro, server W7.

Tried with few projects (12-15 AGridder vst3) which are working well with 1.2 beta 3. (RME 1024 buffer 1 block in AG).
Only the windows size initialization of plugin GUI thing occurs.(#546)
I don't use Vst2 versions anymore because they don't sync with Cubase as I said in #546 too (until you force with AsioGuard)

Such projects in 1.2 beta 4, crash the server machine (2x Xeon 6(12) cores W7 3.ghz), just after it has been loaded by Cubase or at first attempt to open an AGridder plugin interface.

If project successfully loads, Cubase Asio meter becomes completely mad and lights red, server task manager shows memory load but CPU load becomes erratic, jumping around without stabilizing.
AG Panel in the Cubase Machine says every AG plugin is connected to server.

And... it crashes windows on server. (freezes, no blue screen) when I try to open an AGridder.

Maybe a Win7 issue between beta 3 and beta 4 but, with 1.2 beta 4, when I tried to load an remote plugin into an AGridder from a blank Cubase project, it worked.

Here are my logs.

Cheers.

Igor.

[AudioGridderServer_2021-07-30_15-26-44.log](https://github.com/apohl79/audiogridder/files/6908253/AudioGridderServer_2021-07-30_15-26-44.log)
[AudioGridderPlugin_2021-07-30_15-55-52.log](https://github.com/apohl79/audiogridder/files/6908254/AudioGridderPlugin_2021-07-30_15-55-52.log)

**Comment by igbro** [comment]
Digging...

Sand boxing set to on, helps a bit in 1.2 beta 4 server. No crash but project is not playable, Cubase is totally Asio overloaded even when stopped. (1 block@1024)

I'm only using Acustica Acqua plugins with AG and the panel shows weird I/O numbers. (maybe comps sidechain, which I don't use, can explain the extra channels) 

The more Blocks (tried 4-16-24), the more the project is able to play a very few sec before Cubase Asio sticks in Red.
Setting max blocks, makes server computer crash completely just after the 30 blocks selection.

This exact same project plays fluently at 1 block with 1.2 beta 3! ( same 1024 buffer size in my RME)

Thanks.

![Ag Panel](https://user-images.githubusercontent.com/65025818/127679102-182289a0-eb4e-4bc8-a410-98f69107edfd.JPG)

**Reply by apohl79** [reply]
Can you enable tracing? Do you have core dumps?

**Reply by jaacMusic** [reply]
I'm experiencing the same thing in Cakewalk. Previous projects just crash/freeze it. (Windows Server)

**Comment by igbro** [comment]
Hallo Andreas,

I engaged trace.

This time, windows on server crashed when hitting play with the same project in beta 4.

No core dump and server machine crashed so not been able to force to build one.

Since trace file is too big, I sent this via wetransfer @ e47 with this conversation reminder #679

Thanks.

**Comment by jaacMusic** [comment]
I have been experiencing the same issue; I haven't been able to create a dump. My last resort was to downgrade to beta 3 until the issue is fixed. I tried disabling, sandboxing, parallel loading, screen capture setting with no luck. I can't open any of my previous projects that work fine with beta 3.
